<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Welcome::Internal::CommunityWelcomePageWidget</class>
 <widget class="QWidget" name="Welcome::Internal::CommunityWelcomePageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="contentframe">
     <property name="styleSheet">
      <string notr="true">#contentframe{
   border: none 0px;
   border-bottom: 1px solid #c9c9c9;
   margin: 0px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="labsFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="Utils::WelcomeModeLabel" name="labsTitleLabel">
           <property name="text">
            <string>News From the Qt Labs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Utils::WelcomeModeTreeWidget" name="newsTreeWidget" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="sitesFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="Utils::WelcomeModeLabel" name="supportSitesTitleLabel">
           <property name="text">
            <string>Qt Support Sites</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Utils::WelcomeModeTreeWidget" name="supportSitesTreeWidget" native="true"/>
         </item>
         <item>
          <widget class="Utils::WelcomeModeLabel" name="miscSitesTitleLabel">
           <property name="text">
            <string>Qt Links</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Utils::WelcomeModeTreeWidget" name="miscSitesTreeWidget" native="true"/>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::WelcomeModeTreeWidget</class>
   <extends>QWidget</extends>
   <header location="global">utils/welcomemodetreewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Utils::WelcomeModeLabel</class>
   <extends>QLabel</extends>
   <header location="global">utils/welcomemodetreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
